roles/vpn/templates/ipsec.conf.j2

```diff
@@ -27,12 +27,17 @@ conn %default
 
     right=%any
     rightauth=pubkey
-    rightsourceip={{ vpn_network }},{{ vpn_network_ipv6 }}
 {% if local_dns is defined and local_dns == "Y" %}
     rightdns={{ local_service_ip }}
 {% else %}
     rightdns={% for host in dns_servers.ipv4 %}{{ host }}{% if not loop.last %},{% endif %}{% endfor %}{% if ipv6_support is defined and ipv6_support == "yes" %},{% for host in dns_servers.ipv6 %}{{ host }}{% if not loop.last %},{% endif %}{% endfor %}{% endif %}
 {% endif %}
 
-conn ikev2-pubkey
+{% for user in users %}
+
+conn AlgoUser-{{ user }}
```

Comment:
What is the main goal dividing the connections per user?

Reply:
Dividing connections per user assigns exactly one IP address to each user. I use that to create the hosts file (which you saw [here](https://github.com/trailofbits/algo/pull/795#discussion_r177992102)) to enable DNS resolution.

The only reason I did this was because I couldn't figure out a way for dnsmasq to assign a hostname based on the Algo user's name (as mentioned [in my reply to your previously-linked comment](https://github.com/trailofbits/algo/pull/795#discussion_r178202091)). If I could, then this could be reverted and the solution would be much cleaner.